manim/mobject/coordinate_systems.py

```diff
@@ -127,56 +171,55 @@ def input_to_graph_point(self, x, graph):
 class Axes(VGroup, CoordinateSystem):
     def __init__(
         self,
-        x_min=None,
-        x_max=None,
-        y_min=None,
-        y_max=None,
+        x_range=None,
+        y_range=None,
+        x_length=config.frame_width - 2,
+        y_length=config.frame_height - 2,
         axis_config=None,
         x_axis_config=None,
         y_axis_config=None,
-        center_point=ORIGIN,
```

Comment:
Any specific reason you chose to remove the `center_point` parameter? It's very useful imo

Reply:
The changes under `create_axis` make it so that the "`center_point`" is determined dynamically, making it so that defining `center_point` was a bit redundant. Plus, the axes can also be shifted after creation if necessary.


Without `center_point`: 
![image](https://user-images.githubusercontent.com/70682032/115161235-813a6f00-a06a-11eb-86be-90f104c33e60.png)

with the previous default ```center_point = ORIGIN```:
![image](https://user-images.githubusercontent.com/70682032/115161247-9ca57a00-a06a-11eb-94da-731bd3f934f3.png)

Reply:
> The changes under create_axis make it so that the "center_point" is determined dynamically

I see, that's nice

> Plus, the axes can also be shifted after creation if necessary

It has to be done like `axes.shift(target_origin - axes.c2p(0, 0))`. Again, this looks like a bit too much work. I'm not against removing the `center_point` parameter, but if the preference is to drop it, let's at least have a method to shift the axes's origin to some other point. Something like
```py
def move_origin_to(self, new_origin, *other_mobs):
    for mob in [self, *other_mobs]:
        mob.shift(new_origin - self.c2p(0, 0))
```
This could probably be done in a later PR, just leaving my thoughts.

Reply:
I think this would be better considered in a separate PR.